certbot/account.py

```diff
@@ -53,11 +53,15 @@ class Meta(jose.JSONObjectWithFields):
     def __init__(self, regr, key, meta=None):
         self.key = key
         self.regr = regr
-        self.meta = self.Meta(
-            # pyrfc3339 drops microseconds, make sure __eq__ is sane
-            creation_dt=datetime.datetime.now(
-                tz=pytz.UTC).replace(microsecond=0),
-            creation_host=socket.getfqdn()) if meta is None else meta
+        try:
+            creation_hostname = socket.getfqdn()
+        except UnicodeError:
```

Comment:
(Disclaimer: I'm not remotely familiar with this area of the code; I'm just skimming reviews.)

Are there other cases where a `UnicodeError` could be thrown? (E.g., would this same error also be thrown — and replaced with a message about hostname length — if the hostname contains invalid characters?)